hw/top_earlgrey/formal/top_earlgrey_fpv_prim_cfgs.hjson

```diff
@@ -103,7 +103,7 @@
                dut: prim_count_tb
                fusesoc_core: lowrisc:fpv:prim_count_fpv
                import_cfgs: ["{proj_root}/hw/formal/tools/dvsim/common_fpv_cfg.hjson"]
-               build_opts: ["-define ResetValue 0"]
+               build_opts: ["-define ResetValue 0 -define PrimCountFpv"]
```

Comment:
Nit: do you need to specify a value here as well? Above, you set `PrimCountFpv` to 1.

Reply:
Nuts, you're right. Thanks!